### PR TITLE
OZ clone/proxy

### PR DESCRIPTION
**Pull request by drgorillamd**
- Use OZ clone

### CODE REVIEW

**Comment by mejango** [comment]
nice. love it, but needs someone's else's due diligence.

**Review by mejango — APPROVED** [review]

File: contracts/JBTiered721DelegateDeployer.sol [hunk 1]
```diff
@@ -17,8 +19,6 @@ import './JB721GlobalGovernance.sol';
 contract JBTiered721DelegateDeployer is IJBTiered721DelegateDeployer {
   error INVALID_GOVERNANCE_TYPE();
 
-  uint256 constant DEPLOY_BYTECODE_LENGTH = 13;
```
Comment (mejango) [inline contracts/JBTiered721DelegateDeployer.sol:20]: curious where this "13" is in the OZ extension https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/proxy/Clones.sol

Comment (drgorillamd) [reply contracts/JBTiered721DelegateDeployer.sol:20]: Ooh, I think it's a leftover of our cloning pattern, will remove it, good catch

Comment (drgorillamd) [reply contracts/JBTiered721DelegateDeployer.sol:20]: image

So basically, 13 is the equivalent of the 0x0a (red arrows) which say "hey, the bytecode to keep onchain is at the offset 0a", meaning the constructor is the green box only

Comment (mejango) [reply contracts/JBTiered721DelegateDeployer.sol:20]: got it. cool.

Comment (drgorillamd) [reply contracts/JBTiered721DelegateDeployer.sol:20]: (to be more precise, first we copy the code which starts at 0x0a and is 0x2d long in memory, then we return the content of the memory address 0x0 to 0x2d)

**Review by xBA5ED — APPROVED** [review]

**Pull request merged**